### PR TITLE
Fixing translation issue

### PR DESCRIPTION
**Pull request by jguille2**
Fixing no localized strings in objects.js

### CODE REVIEW

**Comment by jguille2** [comment]
Note: Now I've seen there is another problem with "connect arduino to" translation. Code use **%s** and translation files use **%port**.  Only portuguese translations (pt and pt_BR) have added both strings.

I'll see it later.

**Comment by jguille2** [comment]
Hi @bromagosa ... I'm missing something!

I can update all translation files to **%n** and this run fine... but I think it's better to change **objects.js** (**%s** to **%port**). The problem is that this change breaks something (arduino category is not running!). I don't undestand this break.

**Comment by jguille2** [comment]
I found it!!
If I want to use an _InputSlotMorph_ in my block definitions (in _objects.js_), I must declare it in _blocks.js_ before.
We have two options:
1. We can leave **%s** in block definition and change all language files.
2. We can change **%s** to **%port** in _s4a/objects.js_ and add _InputSlotMorph_ definition in _s4a/blocks.js_
   
   ```
     case '%port':
             part = new InputSlotMorph();
             break;
   ```

I vote for the first option. @bromagosa Tell me your opinion and I'll update the PR

**Comment by bromagosa** [comment]
Okay, so now I know what you mean! This translation string is really, really old, and it comes from when we didn't have a "connect Arduino" button. The `%port` input slot was a custom dropdown menu that let you choose among the available serial ports, but caused some trouble as searching for them is asynchronous, and you want the menu to pop up synchronously.

The correct option is the first one, just change `%port` to `%s` in all translation files. `%s` means string, which is what goes inside this slot, so it's the proper placeholder :)

File: snap/s4a/objects.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ SpriteMorph.prototype.initArduinoBlocks = function () {
         only: SpriteMorph,
         type: 'command',
         category: 'arduino',
-        spec: 'connect arduino at %s'
+        spec: 'connect arduino at %s',
+		translatable: true
```
Comment (bromagosa) [inline snap/s4a/objects.js]: `translatable` here refers to translatable to C in the Arduino translation mode. This block is not translatable into C, as the resulting code runs inside the board, so we don't need to connect to it :)

I agree though, it should be called something else... transpilable?

**Comment by jguille2** [comment]
@bromagosa Hi Bernat!
Now this PR is ready to merge, fixing translations of "servo default option" and the block "connect arduino at".

Thanks
Joan

**Comment by bromagosa** [comment]
Thanks! :)

**Pull request merged**